### PR TITLE
MAUI-2551 - Updated UG content for DefaultColumnWidth and ActualWidth of MAUI DataGrid

### PR DESCRIPTION
**Pull request by NirmalKumarYuvaraj**
I have updated the UG content for Column Sizing in MAUI DataGrid.

### CODE REVIEW

File: MAUI/DataGrid/Column-sizing.md [hunk 1]
```diff
@@ -84,6 +84,38 @@ The code below applies the [ColumnWidthMode.Fill](https://help.syncfusion.com/cr
 
 ![Fill column width mode in .NET MAUI DataGrid](Images\column-sizing\maui-datagrid-fill-column-width-mode.png)
 
+## Change the default column width for columns
+
+Based on the value of the [DefaultColumnWidth](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_DefaultColumnWidth) property in [SfDataGrid](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html), [DataGridColumn](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridColumn.html) renders in view.
```
Comment (NeelakandanKannan) [inline MAUI/DataGrid/Column-sizing.md:89]: @NirmalKumarYuvaraj  - Replace with below content,

 If you want to set the common width for all the columns, you can use the [DefaultColumnWidth](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_DefaultColumnWidth) property.

File: MAUI/DataGrid/Column-sizing.md [hunk 2]
```diff
@@ -84,6 +84,38 @@ The code below applies the [ColumnWidthMode.Fill](https://help.syncfusion.com/cr
...
+{% tabs %}
+{% highlight xaml tabtile="MainPage.xaml" %}
+<syncfusion:SfDataGrid  x:Name="dataGrid"
+                        DefaultColumnWidth="120"
+                        ItemsSource="{Binding Orders}" />
+{% endhighlight %}
+{% endtabs %}
+
+## Retrieve the auto-calculated widths of columns
+Using the [ActualWidth](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridColumn.html#Syncfusion_Maui_DataGrid_DataGridColumn_ActualWidth) property, we can get the column's width after auto-fitting.
```
Comment (NeelakandanKannan) [inline MAUI/DataGrid/Column-sizing.md:100]: You can retrieve the width of the columns when it is auto-calculated based on the `ColumnWidthMode` property using the [ActualWidth](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridColumn.html#Syncfusion_Maui_DataGrid_DataGridColumn_ActualWidth) property.

File: MAUI/DataGrid/Column-sizing.md [hunk 3]
```diff
@@ -84,6 +84,38 @@ The code below applies the [ColumnWidthMode.Fill](https://help.syncfusion.com/cr
...
+
+{% tabs %}
+{% highlight xaml tabtile="MainPage.xaml" %}
+<syncfusion:SfDataGrid  x:Name="dataGrid"
+                        DefaultColumnWidth="120"
+                        ItemsSource="{Binding Orders}" />
+{% endhighlight %}
+{% endtabs %}
+
+## Retrieve the auto-calculated widths of columns
```
Comment (NeelakandanKannan) [inline MAUI/DataGrid/Column-sizing.md:99]: Retrieve the auto-calculated width of columns

File: MAUI/DataGrid/Column-sizing.md (hunk 1 above)
Comment (NirmalKumarYuvaraj) [reply MAUI/DataGrid/Column-sizing.md:89]: Updated the content.

File: MAUI/DataGrid/Column-sizing.md (hunk 2 above)
Comment (NirmalKumarYuvaraj) [reply MAUI/DataGrid/Column-sizing.md:100]: Updated the content .

File: MAUI/DataGrid/Column-sizing.md (hunk 3 above)
Comment (NirmalKumarYuvaraj) [reply MAUI/DataGrid/Column-sizing.md:99]: Updated the UG content heading.

**Review by NeelakandanKannan — APPROVED** [review]
The changes are fine.

**Pull request merged**